app/controllers/BaseYesNoAmountController.scala

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
I would break some of the method signatures into separate lines, generally the ones including implicit.

Reply:
Certainly.